docs/source/conf.py

```diff
@@ -219,6 +220,16 @@
 # Output file base name for HTML help builder.
 htmlhelp_basename = 'StackStormDoc'
 
+
+def previous_version(ver):
+    # XXX: on incrementing major version, minor version counter is lost!
+    major, minor = ver.split('.')
```

Comment:
It's dropped by next line. Else in case of 0.999dev we don't have a second separator. Or how do yuo mean?